code/modules/mob/language/generic.dm

```diff
@@ -23,7 +23,7 @@
 	name = LANGUAGE_TCB
 	desc = "A spiritual successor of Esperanto, established in 2404 in Tau Ceti by Ceti intellectuals. Its unique, fully customized alphabet and structure allow it to be spoken even by most alien species. It's the official language of Tau Ceti and has growing traction in diplomatic circles and Universalists across human space."
 	speech_verb = "says"
```

Comment:
Why doesn't speech_verb have to be a list, but all the other ones do?

Reply:
I was fixing this actually lol